### DISCUSSION TITLE
rabbitmq-amqp-java-client amqp 1.0 support

### DISCUSSION THREAD

**Discussion by bilak** [spring-projects/spring-amqp#2918] [Q&A] [answered]
Hey, 
is there a plan to support [rabbitmq amqp 1.0](https://github.com/rabbitmq/rabbitmq-amqp-java-client) library?

**Comment by artembilan** [comment, marked as answer]
There is no such a plan, but contribution is welcome!
I'm not sure yet what benefit we we would gain here in Spring over an API existing in that library.
But again: I'd be happy to review some PR on the matter for upcoming `4.0` generation next year.
This definitely could be as a new module. 
Something like `spring-rabbit-amqp10`?

Converting this into an issue for future consideration.
Thank you!
